plugins/e57/io/E57Writer.cpp

```diff
@@ -184,26 +191,31 @@ void E57Writer::done(PointTableRef table)
     }
     
     // Set bounding boxes on case by case basis
-    if (std::find(m_dimensionsToWrite.begin(),m_dimensionsToWrite.end(),"colorRed") != m_dimensionsToWrite.end())
+    if (Utils::contains(m_dimensionsToWrite, "colorRed"))
     {
         // found color info
         e57::StructureNode colorbox = e57::StructureNode(*m_imageFile);
         for (const std::string& color: {"Red", "Green","Blue"})
         {
-            auto minmax = pdal::e57plugin::getPdalBounds(pdal::Dimension::Id::Red);
-            colorbox.set("color"+color+"Minimum",e57::IntegerNode(*m_imageFile,minmax.first));
-            colorbox.set("color"+color+"Maximum",e57::IntegerNode(*m_imageFile, minmax.second));
+//ABELL - this seems wrong.  Why would we always use red?
```

Comment:
We alsways use Red because Pdal treates all colors the same. But I guess we could change the code and use the actual color.

Reply:
But you're fetching the min/max values of the dimension, right?  Wouldn't those be different (at least potentially) for red, green and blue?  If this isn't what you're doing, maybe a comment is in order.